### ISSUE TITLE
Reproducing WRN-28-10 (SAM) for SVHN dataset

### ISSUE THREAD

**Issue by akshaydp1995** [moskomule/sam.pytorch#1]
I am trying to reproduce the results for WRN-28-10 (SAM) trained on 10-class classification SVHN dataset (Percentage Error 0.99) - https://paperswithcode.com/sota/image-classification-on-svhn  

I'm able to train WRN-28-10 using https://github.com/hysts/pytorch_wrn (Modified the script to incorporate SAM into it)

I'm achieving a test accuracy of 93%. How can I replicate the SOTA Percentage Error 0.99 for WRN-28-10 (SAM). Which hyperparameters do I use?

Any help is appreciated!!

**Comment by moskomule** [comment]
Hi, I'm not the author of SAM, but as far as I know you need

1. higher weight decay rate like 0.001.
2. lower initial learning rate like 0.01
